src/js/components/CheckBox/stories/CheckBoxMaterial.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
```

Comment:
Remove this file. Don’t need to add extra material storybook, we are making it the default theme.

Reply:
Removed